apps/pom.xml

```diff
@@ -109,7 +109,7 @@
         <!-- This snakeyaml version overrides the one normally imported by jackson-dataformat-yaml. -->
         <snakeyaml.version>2.0</snakeyaml.version>
 
-        <jakarta.xml.bind.api.version>2.3.2</jakarta.xml.bind.api.version>
+        <jakarta.xml.bind.api.version>2.3.3</jakarta.xml.bind.api.version>
```

Comment:
Historically, we've hesitated in upgrading these [dependencies due to incompatibility concerns](https://github.com/CMSgov/beneficiary-fhir-data/blob/3e10be192e268cc3d32dbc621cabc2f72b245935/docs/runbooks/how-to-resolve-snyk-bot-prs.md), referenced in the _exclusion_ list as "jakarta.*:* (Jakarta EE)." Personally, I'm unclear if this guidance is outdated or if it really is appropriately pessimistic/conservative in this context. I would think that the ITs/E2E testing we have in place would uncover any serious compatibility problems, and if this change set should pass, it should be generally safe to accept it as-is.

Looking for input from @brianburton @lsmitchell @mackec1 for warmer, _fuzzier_ feelings.

Reply:
that would be my take as well....this was a point release, which almost always deals with bug fixes. I'd be inclined to perform more stringent if it were a minor rev (and certainly REALLY stringent on a major rev) release.